(customizing-clerk/localization) Document how to use pipes and modifiers (#3370)
clerkinc/clerk-docs: docs/guides/customizing-clerk/localization.mdx

  })
  ```
</If>

### Pipe syntax and modifiers

Clerk localization strings support a small template language. Some values will use `{{ variable | modifier(...args) }}` to pass a runtime value through one or more named formatters. Here's what each part means:

> - `variable` — the runtime value (like in the example, a `Date` passed as `date`).

Reviewer: The "like in" syntax feels a little conversational to me. Could we maybe change it to "for example" or such as?

> - `|` — the pipe operator; it sends the value into the modifier to its right.

Reviewer: ```suggestion
- `|` — the pipe operator; it passes the value to the modifier on its right.
```

Just a suggestion, but feels a bit smoother to me with "passes" as more common in technical terms.

- `modifier(...)` — a named formatter with optional arguments (like in the example, the `timeString` formatter with the `'en-US'` locale as an argument).

For example: `"Yesterday at {{ date | timeString('en-US') }}"` would take the `date` and apply the `timeString` formatter with the `'en-US'` locale to get the time string in English, like `'12:00 PM'`.

#### Chaining pipes

You can chain pipes so each step receives the result of the previous one.

> For example: `{{ date | weekday('en-US','long') | titleize }}` would take the `date` and apply the `weekday` formatter with the `'en-US'` locale and `'long'` style ('Monday' instead of 'Mon') to get a long English weekday name. Then, `titleize` would ensure the long English weekday name is properly capitalized. Some countries lowercase weekday names while others capitalize them. Overall, a rendered output would be something like `'Monday'` instead of `'Mon'` or `'monday'`.

Reviewer: ```suggestion
For example: `{{ date | weekday('en-US','long') | titleize }}` would take the `date` and apply the `weekday` formatter with the `'en-US'` locale and `'long'` style ('Monday' instead of 'Mon') to get a long English weekday name. Then, `titleize` would ensure the long English weekday name is properly capitalized. This is useful because some locales return lowercase weekday names while others capitalize them, so the final output would consistently be something like `'Monday'` rather than `'Mon'` or `'monday'`.
```

Just a suggestion, but think "some locales" instead of "some countries" work a bit better? It feels a bit more precise, since capitalization conventions are generally language/locale-specific.
